### ISSUE TITLE
Underscore character not seen on integrated terminal

### ISSUE THREAD

**Issue by singh-sumit** [microsoft/vscode#129142]
Issue Type: <b>Bug</b>

I am using VSCode 1.58; the problem is underscore is not seen in terminal, **rather it is replaced by space** as seen in screenshot over below.
![img](https://imgur.com/uHwxiLU.png)

- Also the **folder/file name with "_" is replaced by space;** when i cd into that directory.

## **But in external terminal of Ubuntu, result is clear**;
![img](https://imgur.com/omYWwYZ.png)

### My VSCode configurations:
Version: 1.58.2
Commit: c3f126316369cd610563c75b1b1725e0679adfb3
Date: 2021-07-14T22:30:16.440Z
Electron: 12.0.13
Chrome: 89.0.4389.128
Node.js: 14.16.0
V8: 8.9.255.25-electron.0
OS: Linux x64 5.8.0-63-generic

VS Code version: Code 1.58.2 (c3f126316369cd610563c75b1b1725e0679adfb3, 2021-07-14T22:30:16.440Z)
OS version: Linux x64 5.8.0-63-generic
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-7500U CPU @ 2.70GHz (4 x 3500)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>opengl: enabled_on<br>rasterization: disabled_software<br>skia_renderer: enabled_on<br>video_decode: disabled_software<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|1, 2, 2|
|Memory (System)|11.60GB (1.99GB free)|
|Process Argv|--unity-launch --crash-reporter-id eb7794e9-79fd-4dc8-a667-e822057037c2|
|Screen Reader|no|
|VM|0%|
|DESKTOP_SESSION|ubuntu|
|XDG_CURRENT_DESKTOP|Unity|
|XDG_SESSION_DESKTOP|ubuntu|
|XDG_SESSION_TYPE|x11|
</details><details><summary>Extensions (10)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-sqlite|ale|0.13.0
vscode-django|bat|1.6.0
python|ms-|2021.7.1053846006
vscode-pylance|ms-|2021.7.4
jupyter|ms-|2021.8.1046824664
LiveServer|rit|5.6.1
markdown-preview-enhanced|shd|0.5.22
vscodeintellicode|Vis|1.2.14
vscode-icons|vsc|11.5.0
jinja|who|0.0.8

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
pythonvspyt700cf:30270857
pythonvspyt602:30300191
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythonvspyt639:30300192
pythontb:30283811
vspre833:30321513
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
pythondataviewer:30285071
vscus158:30321503
pythonvsuse255:30340121
vscod805cf:30301675
pythonvspyt200:30340761
vscextlang:30333561
binariesv615:30325510
pythonvssor306cf:30340756
bridge0708:30335490

```

</details>

**Comment by IllusionMH** [comment]
Looks like duplicate of #123077

Fix should be available in Insiders builds https://code.visualstudio.com/insiders/

Is it reproducible for you in Insiders builds?

**Comment by singh-sumit** [comment]
Yes, it is working fine on insiders builds.

**Comment by IllusionMH** [comment]
/duplicate

Fix should be in stable 1.59 release. In the meantime you can use Insiders builds or try workaround from linked issue `"terminal.integrated.gpuAcceleration": "on"`
